src/test/java/com/ontotext/trree/plugin/rdfrank/TestPluginRDFRankWithRDFStarData.java

```diff
@@ -0,0 +1,315 @@
+package com.ontotext.trree.plugin.rdfrank;
+
+import com.ontotext.test.TemporaryLocalFolder;
+import com.ontotext.test.utils.Utils;
+import com.ontotext.trree.OwlimSchemaRepository;
+import com.ontotext.trree.entitypool.EntityPoolConnection;
+import com.ontotext.trree.entitypool.EntityType;
+import org.eclipse.rdf4j.model.*;
+import org.eclipse.rdf4j.model.impl.SimpleValueFactory;
+import org.eclipse.rdf4j.query.*;
+import org.eclipse.rdf4j.repository.RepositoryConnection;
+import org.eclipse.rdf4j.repository.RepositoryException;
+import org.eclipse.rdf4j.repository.sail.SailRepository;
+import org.eclipse.rdf4j.rio.RDFFormat;
+import org.eclipse.rdf4j.sail.SailConnection;
+import org.eclipse.rdf4j.sail.SailException;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.*;
+
+import static org.junit.Assert.*;
+
+@RunWith(Parameterized.class)
+public class TestPluginRDFRankWithRDFStarData {
+	private SimpleValueFactory VF = SimpleValueFactory.getInstance();
```

Comment:
The class has a lot of methods similar or the same as TestPluginRDFRank. Please compare the two files and extract the common code in a base class to avoid code duplication.